lib/libutee/tee_api_operations.c

```diff
@@ -1328,6 +1332,7 @@ TEE_Result TEE_AEInit(TEE_OperationHandle operation, const void *nonce,
 		goto out;
 
 	operation->ae_tag_len = tagLen / 8;
+	operation->info.digestLength = operation->ae_tag_len;
```

Comment:
since this change, `ae_tag_len` field in useless:
```
struct __TEE_OperationHandle {
	(...)
	uint32_t ae_tag_len;	/*
				 * tag_len in bytes for AE operation else unused
				 */
```
Functions should fetch straight `operation->info.digestLength`.

Reply:
Yes, this makes this field useless now. I'll get rid of it.